features/shared-objects/constants.js

```diff
@@ -1,3 +1 @@
-module.exports = {
-  url: 'http://localhost:8080'
-};
\ No newline at end of file
+module.exports = {};
```

Comment:
What might go in here? Unless we have something in mind this should possibly be removed

Reply:
Yeah now that I think about it we can probably just define constants in the files they're needed or in page-objects.  I'll delete this file